src/commcare_cloud/environment/schemas/meta.py

```diff
@@ -5,5 +5,5 @@ class MetaConfig(jsonobject.JsonObject):
     _allow_dynamic_properties = False
     deploy_env = jsonobject.StringProperty(required=True)
     env_monitoring_id = jsonobject.StringProperty(required=True)
-    users = jsonobject.StringProperty(required=True)
+    users = jsonobject.ListProperty(required=True)
```

Comment:
I left a comment directly on the commit: https://github.com/dimagi/commcare-cloud/commit/70e7dba646145367e1057992b79c44e017f34d79#r29236121